spec.md

```diff
@@ -135,8 +144,8 @@ Subsections:
 - Describe caveats.
 - If using an external API generator (like go-doc, js-doc, or so on), point to an external `API.md` file. This can be the only item in the section, if present.
 
-#### Contribute
-**Status**: Required.  
+### Contribute
+**Status**: Required.
```

Comment:
Trailing spaces are REQUIRED by Markdown? That's a ... horrifying ... design decision. But certainly, I'll avoid removing them.